src/app/_lib/firebase-config.ts

```
@@ -0,0 +1,20 @@
+// lib/firebase-config.ts
+import { initializeApp, getApps } from "firebase/app";
+import { getMessaging, isSupported, Messaging } from "firebase/messaging";
+
+const firebaseConfig = {
+  apiKey: process.env.NEXT_PUBLIC_FIREBASE_API_KEY!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)